src/sentry/api/serializers/models/project.py

```diff
@@ -325,11 +317,28 @@ def serialize(self, obj, attrs, user):
                     'digests:mail:maximum_delay',
                     digests.maximum_delay,
                 ),
-                'subjectPrefix':
-                attrs['options'].get('mail:subject_prefix'),
+                'allowedDomains':
+                attrs['options'].get(
+                    'sentry:origins', ['*']),
+                'resolveAge':
```

Comment:
This was a dcramer change, probably for frontend consumption reasons....
it's unclear when things should be in the `options` hash i.e. `digestsMinDelay`